benchmark/Streamly/Benchmark/Data/Stream/Generate.hs

```diff
@@ -151,6 +164,40 @@ o_1_space_generation value =
         , benchIOSrc fromSerial "absTimes" $ absTimes value
         ]
     ]
+#else
```

Comment:
Why aren't these benchamrks common to both Prelude/Data.Stream cases? Why are these only for Data.Stream case?

Reply:
bcoz all of them are polymorphic t m a.

Reply:
I am talking about the `else` case. The else case should be common and not conditional.